dev/io.openliberty.mcp.internal_fat/fat/src/io/openliberty/mcp/internal/fat/tool/ToolTest.java

```diff
@@ -40,6 +41,10 @@
 @RunWith(FATRunner.class)
 public class ToolTest extends FATServletClient {
 
+    private static final String ACCEPT_HEADER = "application/json, text/event-stream";
+    private static final String mcpProtocolHeader = "MCP-Protocol-Version";
+    private static final String mcpProtocolVersion = "2025-06-18";
```

Comment:
```suggestion
    private static final String ACCEPT_HEADER = "application/json, text/event-stream";
    private static final String MCP_PROTCOL_HEADER = "MCP-Protocol-Version";
    private static final String MCP_PROTCOL_VERSION = "2025-06-18";
```

Capitalize constants.

Reply:
Missed those 2, sorry